core/src/cpus/cortex_m.zig

```diff
@@ -15,6 +20,42 @@ pub fn disable_interrupts() void {
     asm volatile ("cpsid i");
 }
 
+pub fn enable(comptime interrupt: anytype) void {
+    const interrupt_name = @tagName(interrupt);
+    if (@hasField(Interrupt, interrupt_name)) {
+        const num = @intFromEnum(@field(Interrupt, interrupt_name));
+        if (num >= 0) {
+            peripherals.nvic.unmask(num);
+        } else {
+            @compileError("can't enable exception: " ++ interrupt_name);
+        }
+    } else {
+        @compileError("interrupt not found: " ++ interrupt_name);
+    }
+}
+
+pub fn disable(comptime interrupt: anytype) void {
+    const interrupt_name = @tagName(interrupt);
+    if (@hasField(Interrupt, interrupt_name)) {
```

Comment:
could probably abstract these checks out to some sort of 'interrupt is valid' comptime function to reduce duplicated logic.